tap-gui/tap-gui-rbac/setup-tap-gui-rbac-eks.md

```diff
@@ -0,0 +1,157 @@
+
+## <a id="configure-cluster"></a> Configure the Kubernetes cluster with the OIDC provider
+
+These steps will help you to configure the cluster with the OIDC provider's credentials.
+
+1. Create a file with the contents below and name it `rbac-setup.yaml`. The following example applies to EKS clusters.
+
+```console
+apiVersion: eksctl.10/vialphas
+kind: ClusterConfig
+metadata:
+    name: "CLUSTER-NAME"
+    region: "AWS-REGION"
+identitvProviders:
+  - name: autho
+    type: oidc
+    issuerUrl: "ISSUER-URL"
+    clientId: "CLIENT-ID"
+    usernameClaim: email
+```
+Where:
+   - `CLUSTER-NAME` is the cluster name for your EKS cluster
+   - `AWS-REGION` is the AWS region of the EKS cluster
+   - `ISSUER-URL` is the Issuer URL you obtained while setting up the OIDC provider
+   - `CLIENT-ID` is the Client ID you obtained while setting up the OIDC provider
+
+
+2. Using `eksctl`, execute the following command:
+
+```console
+eksctl associate identityprovider -f rbac-setup.yaml
+```
+
+3. To check if the association of the OIDC provider with the EKS cluster was successful, run the following command
+
+```console
+eksctl get identityprovider --cluster `CLUSTER-NAME`
+```
+
+The output should state `ACTIVE` in the STATUS column.
+
+
+## <a id="configure-tap-gui"></a> Configure the Tanzu Application Platform GUI
+
+The next step is to configure visibility of the remote cluster in Tanzu Application Platform GUI. 
+
+1. Make sure you added an `auth` section to the `app_config` file taht Tanzu Application Platform GUI uses. In the example for Auth0, copy this YAML content into `tap-gui-values.yml`:
+
+
+```console
+auth:
+    environment: development
+    providers:
+        autho:
+            development:
+                clientId: "CLIENT-ID"
+                clientSecret: "CLIENT-SECRET"
+                domain: "ISSUER-URL"
+```
+Where:
+
+   - `CLIENT-ID` is the Client ID you obtained while setting up the OIDC provider
+   - `ISSUER-URL` is the Issuer URL you obtained while setting up the OIDC provider
+   - `CLIENT-SECRET` is the Client Secret you obtained while setting up the OIDC provider
+
+
+2. You must also add a `kubernetes` section to the `app_config` file that Tanzu Application Platform GUI uses. This section must have an entry for each cluster that has resources to view.
+
+To do so, copy this YAML content into `tap-gui-values.yml`:
+
+
+```console
+kubernetes:
+    serviceLocatorMethod:
+        type: 'multiTenant'
+    clusterLocatorMethods:
+      - type: 'config'
+        clusters:
+          - name: "CLUSTER-NAME"
+            url: "CLUSTER_URL"
+            authProvider: oidc
+            oidcTokenProvider: auth0
+            skipTLSVerify: true
+            skipMetricsLookup: true
```

Comment:
fix indentation
use YAML syntax highlighting and fix indentation
~~~suggestion
          authProvider: oidc
          oidcTokenProvider: auth0
          skipTLSVerify: true
          skipMetricsLookup: true
~~~

Reply:
Thanks